e2e_test_support_files/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+
+services:
+  gateway:
+    image: gateway
+    restart: "unless-stopped"
+    depends_on:
+      - frontend
+      - backend
+    ports:
+      - "8080:80"
```

Comment:
Is there a reason this is exposed on port 8080 rather than 80?

Reply:
I just reused the `.env.example` file. which uses the `8080` port.